libr/core/cmd.c

```diff
@@ -1456,14 +1456,13 @@ static int cmd_kuery(void *data, const char *input) {
 			}
 			cur_cmd = r_str_ndup (out, cur_pos - out);
 
-			r_cons_printf ("\n\n\"%s\" : [", cur_cmd);
+			r_cons_printf ("\n\"%s\" : [", cur_cmd);
 
 			next_cmd = r_str_newf ("anal/%s/*", cur_cmd);
```

Comment:
this is leaking

Reply:
you must call `free (next_cmd)` before overwriting the previous value and `R_FREE (next_cmd)` after the `while(*out){` loop